src/encoded/static/components/experiment_series.js

```diff
@@ -120,43 +125,30 @@ const experimentTableColumns = {
+        getValue: (experiment) => _.uniq(experiment.selectedFiles.map((f) => f.mapped_run_type).filter((l) => l)).join(', '),
+    },
+
     readDepth: {
         title: <div>Read depth<Tooltip trigger={<i className="icon icon-question-circle" />} tooltipId="qc-report-nrf" css="tooltip-home-info">Number of mapped reads passing quality control filtering. Minimum read depth is 5M. For broad histone marks, acceptable read depth is &gt;35M and recommended read depth is &gt;45M. For other targets, acceptable read depth is &gt;10M and recommended read depth is &gt;20M.</Tooltip></div>,
         display: (experiment, meta) => {
-            const low = 5000000;
-            let minimal = 10000000;
-            let recommended = 20000000;
-            if (experiment.target && experiment.target.investigated_as.includes('broad histone mark')) {
-                minimal = 35000000;
-                recommended = 45000000;
-            }
-            const filteredFiles = experiment.files.filter((f) => f.biological_replicates.includes(meta.bioRepNum));
-            const qualityMetrics = filteredFiles.map((f) => f.quality_metrics).reduce(
-                (flatQualityMetrics, qms) => flatQualityMetrics.concat(qms), []
-            );
-            let qm = [];
-            if (experiment.target && ['H3K9me3-human', 'H3K9me3-mouse'].includes(experiment.target.name)) {
-                qm = _.uniq(qualityMetrics.filter((q) => q.processing_stage === 'unfiltered').map((q) => q.mapped / ((q.read1 && q.read2) ? 2 : 1)).filter((q) => q));
-            } else {
-                qm = _.uniq(qualityMetrics.filter((q) => q.processing_stage === 'filtered').map((q) => q.total / ((q.read1 && q.read2) ? 2 : 1)).filter((q) => q));
-            }
-            if (qm && qm.length > 1) {
-                return <td key="readDepth" className="qc-report">?</td>;
-            }
-            if (qm && qm.length === 1) {
-                let qcClass = 'qc-report';
-                if (qm[0] < low) {
-                    qcClass = qcClass.concat(' qc-report--error');
-                } else if (qm[0] >= low && qm[0] < minimal) {
-                    qcClass = qcClass.concat(' qc-report--not-compliant');
-                } else if (qm[0] >= minimal && qm[0] < recommended) {
-                    qcClass = qcClass.concat(' qc-report--warning');
-                } else {
-                    qcClass = qcClass.concat(' qc-report--ideal');
+            if (experiment.selectedAnalysis && experiment.selectedAnalysis.quality_metrics_report) {
+                const metricObj = getQualityMetricsByReplicate(
+                    experiment.selectedAnalysis.quality_metrics_report,
+                    'Read depth',
+                    meta.bioRepNum
+                );
+                if (metricObj && metricObj.metric) {
+                    return <td key="readDepth" className={qcModifierMap[metricObj.quality] || qcBlock}>{(metricObj.metric / 1000000).toFixed(2).concat('M')}</td>;
```

Comment:
Do you need the `key` attribute here? You normally only need these within a `.map` loop. I’d be surprised if you needed them here and in the other table properties.

Reply:
Removed keys and don't get errors on console. So fine to remove.